<launch>
    <node pkg="gpd" type="gpd_interaction" name="gpd_interaction">
        <!--launch-prefix="xterm -e gdb args">-->
        <param name="/estimated_poses" value="/estimated_poses"/>
        <param name="/outgoing_poses" value="/grasp_pose"/>
</node>
</launch>
